functions/index.js

```diff
@@ -52,9 +51,10 @@ exports.collectParkData = functions.pubsub
             }
             console.log(currentEntryData);
             // eslint-disable-next-line max-len
-            const currentWaitTime = attraction.queue && attraction.queue["STANDBY"] && attraction.queue["STANDBY"].waitTime ? attraction.queue["STANDBY"].waitTime : 0;
+            const currentWaitTime = attraction.queue && attraction.queue["STANDBY"] && attraction.queue["STANDBY"].waitTime ?
+                                    attraction.queue["STANDBY"].waitTime : 0;
             // eslint-disable-next-line max-len
```

Comment:
There were a few lines that ended up with too many characters despite me breaking them up the best I could. There was an 80 character max I think per line?

Reply:
You can return right before each operator and it's pretty standard code style

```
const currentWaitTime = attraction.queue 
&& attraction.queue["STANDBY"] 
&& attraction.queue["STANDBY"].waitTime 
     ? attraction.queue["STANDBY"].waitTime 
     : 0;

```

Reply:
You can also use something called "optional chaining", I'm assuming that you're using the multiple && operators to ensure you don't get a null pointer error. 



```
const currentWaitTime = attraction?.queue?["STANDBY"]?.waitTime || 0 ;
```

Reply:
Btw you'd have to verify how I did the optional chaining on the array access, I'm not 100% sure on the syntax off the top of my head

Reply:
Yes I think I've tried optional chaining in this app a couple different times but it wasn't working for some reason. I'll have to give it another go, I know it would be a lot cleaner.